initial-setup.spec

```diff
@@ -101,7 +101,7 @@ rm -rf %{buildroot}
 
 %files gui
 %{_libexecdir}/%{name}/initial-setup-graphical
-%{python3_sitelib}/initial_setup/gui/*
+%{python3_sitelib}/initial_setup/gui
```

Comment:
I suggest keeping the slash. That way it is more obvious to the reader that it is a recursively included directory.

```suggestion
%{python3_sitelib}/initial_setup/gui/
```

Reply:
Sure, lets do that! :)